torch/distributed/rpc/internal.py

```diff
@@ -49,6 +50,14 @@ def _tensor_reducer(self, obj):
         tensor_index = len(_thread_local_tensor_tables.send_tables) - 1
         return (_InternalRPCPickler._tensor_receiver, (tensor_index,))
 
+    @classmethod
+    def _rref_receiver(cls, rref_fork_data):
+        return dist.rpc.RRef._deserialize(rref_fork_data)
+
+    def _rref_reducer(self, obj):
```

Comment:
We expect `obj` to be an RRef? Shall we explicitly call it rref here? 

Same for the `_tensor_reducer` above.

Reply:
sounds good